src/components/Chatbar/components/Conversation.tsx

```diff
@@ -108,7 +109,14 @@ export const ConversationComponent = ({ item: conversation, level }: Props) => {
   const isReplay = useAppSelector(
     ConversationsSelectors.selectIsReplaySelectedConversations,
   );
-  const folders = useAppSelector(ConversationsSelectors.selectFolders);
+  const folders = useAppSelector((state) =>
+    ConversationsSelectors.selectFilteredFolders(
+      state,
+      getMyItemsFilters(),
+      '',
+      true,
```

Comment:
Its not good to pass booleans as arguments
Is it possible to change just arguments to object with this arguments inside?

Reply:
if we pack argument into object memorization won't work, because it always will be new object/link

Reply:
We need to avoid such things, but let's go with what we have for now